### DISCUSSION TITLE
Disable: Get Windows updates as soon as they're available for your device

### DISCUSSION THREAD

**Discussion by nadinoob** [valinet/ExplorerPatcher#1816] [General]
And you'll have no issues with EP, at least not those that will be patched inside a few hours or days!

**Comment by pyrates999** [comment]
Also, in the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

**Comment by gcormier** [comment]
Where is this option?

**Reply by pyrates999** [reply]
https://www.elevenforum.com/t/enable-or-disable-get-latest-updates-as-soon-as-available-in-windows-11.13821/

**Reply by gcormier** [reply]
Ah, thanks - I already had it turned off and ExplorerPatcher still broke a few days ago.

**Reply by pyrates999** [reply]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.
